src/components/ComboBox/ComboBox.tsx

```diff
@@ -86,13 +86,13 @@ class ComboBox extends React.PureComponent<Props, State> {
     if (this.props.sortEntity) {
       this.props.sortEntity(field, order, sortBy);
     }
-  };
-  
+  }
+
   componentDidMount() {
     const { items } = this.props;
     if (items && items[0]) {
-      const { type = "" } = items && items[0];
-      if (type !== "Tabuler") {
+      const { type = '' } = items && items[0];
+      if (type !== 'Tabuler') {
```

Comment:
@Farhan-emgage I know this is not related to this issue, but can you please correct the spelling if this itemType option. The correct spelling is "Tabular". You'll also need to check our other repo's to see if ComboBox is used with itemType set to "Tabuler" and change it to "Tabular". Thanks!

Reply:
@weegrom Ok i will check

Reply:
@Farhan-emgage please provide an update as to when this request will be completed.